drydock/templates/drydock/k8s/lifecycle/cms.yml

```diff
@@ -12,16 +12,37 @@ spec:
               exec:
                 command:
                 - "sleep"
-                - "30"
+                - "15"
```

Comment:
I still want to challenge this preStop hook. Why is that we need this? Why isn't the uWSGI finish routine not enough?

Reply:
@Ian2012 was performing a few tests related to this feature, we should get good input from there soon.

Reply:
After the tests, the behavior is slightly better. We decided to keep the `preStop` hook with a 15 seconds sleep period